### PR TITLE
docs: bcrypt reference config

### PR DESCRIPTION
**Pull request by vinckr**
Added short explanation to config.js
Added maximum values to config.schema.json

Is there some way I can test building the config?

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

**Comment by vinckr** [comment]
https://github.com/ory/hydra/issues/2077

File: docs/scripts/config.js [hunk 1]
```diff
@@ -187,6 +187,11 @@ flag: \`${config.projectSlug} --config path/to/config.yaml\`.
 Config files can be formatted as JSON, YAML and TOML. Some configuration values support reloading without server restart.
 All configuration values can be set using environment variables, as documented below.
 
+This reference configuration documents all keys, also deprecated ones!
```
Comment (aeneasr) [inline docs/scripts/config.js:190]: @vinckr this needs to be changed here: https://github.com/ory/docs/blob/master/scripts/config.js#L164

File: .schema/config.schema.json [hunk 2]
```diff
@@ -743,7 +743,8 @@
                   "type": "integer",
                   "description": "Sets the BCrypt cost. The higher the value, the more CPU time is being used to generate hashes.",
                   "default": 10,
-                  "minimum": 8
+                  "minimum": 4,
```
Comment (aeneasr) [inline .schema/config.schema.json:746]: I think the minimum is correct?

File: docs/scripts/config.js (hunk 1 above)
Comment (vinckr) [reply docs/scripts/config.js:190]: https://github.com/ory/docs/pull/326

So this change in /hydra/docs is not necessary then right?

Comment (aeneasr) [reply docs/scripts/config.js:190]: yes

**Pull request merged**